[BE] typing for decorators - distributed/_tensor/ops/utils

aorenste opened the pull request:
Test Plan: unit tests

Differential Revision: D62302679




cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

kwen2501 reviewed (approved):
The lint issue looks real. Can you please fix it so that the change can land? Thanks!

aorenste committed:
[BE] typing for decorators - distributed/_tensor/ops/utils (#142139)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/142139

Test Plan: unit tests

Differential Revision: D62302679

oulgen commented:
@pytorchbot merge
